code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -84,6 +84,15 @@
 	burntime = 20
 	// TODO:  Special ash subtype that looks like charred chair legs
 
+/obj/structure/stool/bed/chair/wood/attack_animal(mob/living/simple_animal/user)

```

Comment:
❄️ 

This would better be handled by parent and governed by a var for what drops when it's destroyed and how much, then you don't have to snowflake stuff like this out for every single time you have a new subtype.